README.md

```diff
@@ -25,10 +25,37 @@ The release page includes the full Changelog but you may also see the [changelog
 
 ## Build requirements
 * Visual Studio 2017 or newer
```

Comment:
We should change that to say "... build tools" (which is a separate package, for example pre-installed in several CI environments) or specify builds; this is not related to this PR but _could_ be done together.

Reply:
The pre-installed build tools I see in AppVeyor are all VS...
What does "... build tools" mean? Does that mean "Visual Studio 2017 or newer `build tools`"?
Or you you wanted something different like the following?
```
C++ build tools (MS BuildTools, Visual Studio 2017 or newer)
```

Reply:
I mean - you don't need the full Visual Studio installed, the [Build Tools for C++](https://visualstudio.microsoft.com/de/visual-cpp-build-tools/) (which are included in Visual Studio) are enough.

Reply:
Alright, its the latter, I'll update it :ok_hand: